bullet.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+### THIS CODE MADE FOR STAY IN TARKOV AND LICENSED UNDER MIT
+
+SIT_VERSION="${SIT:=latest}"
+HEADLESS="${HEADLESS:-}"
+FORCE="${FORCE:-}"
+echo "Stay In Tarkov Docker"
+echo "github.com/StayInTarkov"
+
+if [ ! -e "/opt/server/version" ] || [ ! -z "$FORCE" ]; then
+  if [ -d "/opt/srv" ]; then
+    start=$(date +%s)
+    echo "Started copying files to your volume/directory.. Please wait."
+    cp -r /opt/srv/* /opt/server/
+    rm -r /opt/srv
+    end=$(date +%s)
+    echo "Files copied to your machine in $(($end-$start)) seconds."
+    echo "Starting the server to generate all the required files.. Please wait."
+    cd /opt/server
+    chown $(id -u):$(id -g) ./* -Rf
+    EXT_IP=$(curl -4 ipv4.icanhazip.com)
```

Comment:
Seems fine, though would you be open to passing this in as an environment variable (because the hosting environment knows what its IP is) and defaulting to the `curl` call otherwise?